modules/lib/src/main/resources/assets/js/app/inputtype/ui/selector/image/ImageEditor.ts

```diff
@@ -752,6 +754,32 @@ export class ImageEditor
             }
         });
 
+        const editContainerButtonsResponsiveHandler = () => {
+            const item = new ResponsiveItem(this);
+
+            if(item.isInRangeOrSmaller(ResponsiveRanges._360_540)) {
+                applyButton.setLabel('Ok');
+                applyButton.setTitle('Ok');
+                applyButton.addClass('ok-button');
```

Comment:
You should not need a separate CSS class. If you want to apply certain CSS formatting for a specific resolution, this can be done directly in CSS. In this case, since your `ResponsiveItem` is the `ImageEditor` div, it will have a CSS class like '`_360-540`' assigned to it, which shows current width range of the element.

So, add _once_ a CSS class to the "Apply" button when it's created, like 'apply-button':

`const applyButton: Button = new Button(i18n('button.apply')).addClass('apply-button');`

Then, in CSS (`image-editor.less`) you can simply do:

```CSS
.image-editor {
  padding: 0;
  &._360-540 {
     .apply-button {
         padding: 0 6px;
     }  
  }
}
```